### ISSUE TITLE
Some modifiers do not work with css modules

### ISSUE THREAD

**Issue by docccdev** [docccdev/evokit#4]
Some modifiers do not work with css modules, if `className` only `hash` for production.

Example settings webpack `css-loader`:

``` jsvascript
{
    test: /\.css$/i,
    loader: 'css-loader',
    options: {
        modules: {
            localIdentName: isProd ? '[hash:base64:5]' : '[name]__[local]--[hash:base64:5]'
        }
    }
}
```

This modifiers don't work:

- `grid-indent` with `grid-divider`
- `list-indent` with `list-divider`
- `text-wrap` with `text-lheight`

Problem in specific css selectors:
``` css
&[class*='...']
```

**Comment by docccdev** [comment]
`list-indent` fixed in `evokit-list@3.2.0`

**Comment by docccdev** [comment]
`grid-indent` fixed in `evokit-grid@3.3.0`

**Comment by docccdev** [comment]
fixed in `evokit-text@3.2.0`
